src/Illuminate/Database/Schema/Blueprint.php

```diff
@@ -1000,11 +1000,17 @@ public function boolean($column)
      * Create a new enum column on the table.
      *
      * @param  string  $column
-     * @param  array  $allowed
+     * @param  string|array  $allowed
      * @return \Illuminate\Database\Schema\ColumnDefinition
      */
-    public function enum($column, array $allowed)
+    public function enum($column, string|array $allowed)
```

Comment:
Union types are PHP 8+ I believe? Laravel 8 supports PHP 7.4.

Reply:
Ya, sorry.  Good catch.  I will loosen the signature like the other methods.